### PR TITLE
Save model diagram

### PR DESCRIPTION
**Pull request by twolodzko**
Most of the changes are inherited from #14

### CODE REVIEW

File: neptune_tensorflow_keras/impl/__init__.py [hunk 1]
```diff
@@ -95,21 +100,36 @@ class NeptuneCallback(Callback):
...
+        run: Run,
+        base_namespace: str = "training",
+        log_model_diagram: bool = False,
+    ):
         super().__init__()
 
+        self.log_model_diagram = log_model_diagram
+
         expect_not_an_experiment(run)
         verify_type("run", run, Run)
```
Comment (Raalsky) [inline neptune_tensorflow_keras/impl/__init__.py:114]: Rember to verify input type of `log_model_diagram`

File: neptune_tensorflow_keras/impl/__init__.py [hunk 2]
```diff
@@ -95,21 +100,36 @@ class NeptuneCallback(Callback):
...
-    def __init__(self, run: Run, base_namespace: str = "training"):
+    def __init__(
+        self,
+        run: Run,
+        base_namespace: str = "training",
+        log_model_diagram: bool = False,
+    ):
         super().__init__()
 
+        self.log_model_diagram = log_model_diagram
```
Comment (Raalsky) [inline neptune_tensorflow_keras/impl/__init__.py:111]: Should be internal property `self._log_model_diagram` and placed after parameters verification

Comment (twolodzko) [reply neptune_tensorflow_keras/impl/__init__.py:111]: Done

File: CHANGELOG.md [hunk 3]
```diff
@@ -14,6 +14,11 @@
 
 - We fixed the dependencies for TensorFlow to version >= 2.0.0. ([#14](https://github.com/neptune-ai/neptune-tensorflow-keras/pull/14))
 
+### Added
```
Comment (Raalsky) [inline CHANGELOG.md:17]: There is already created `Added` section :wink:

File: neptune_tensorflow_keras/impl/__init__.py [hunk 4]
```diff
@@ -125,9 +146,12 @@ def _log_metrics(self, logs, category: str, trigger: str):
                 pass
 
     def on_train_begin(self, logs=None):  # pylint:disable=unused-argument
-        self._metric_logger["model/summary"] = _model_summary_file(self.model)
+        self._model_logger["summary"] = _model_summary_file(self.model)
         self._metric_logger["model/optimizer_config"] = self.model.optimizer.get_config()  # it is a dict
```
Comment (Raalsky) [inline neptune_tensorflow_keras/impl/__init__.py:150]: Let's update it accordingly with `self._model_logger`

Comment (twolodzko) [reply neptune_tensorflow_keras/impl/__init__.py]: Done.

**Review by Raalsky — APPROVED** [review]

**Pull request merged**